File: .sdk/tm/ts/test/exists.test.ts

import { test, describe } from 'node:test'
import { equal } from 'node:assert'


import { KeanuWhoaSDK } from '..'


describe('exists', async () => {

  test('test-mode', async () => {
    const testsdk = await KeanuWhoaSDK.test()
    equal(null !== testsdk, true)
  })

})

File: lua/test/exists_test.lua
-- ProjectName SDK exists test

local sdk = require("keanu-whoa_sdk")

describe("KeanuWhoaSDK", function()
  it("should create test SDK", function()
    local testsdk = sdk.test(nil, nil)
    assert.is_not_nil(testsdk)
  end)
end)
